keyboards/puck/puck.h

```diff
@@ -0,0 +1,17 @@
+#ifndef PUCK_H
+#define PUCK_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B \
+) { \
+    { K00, K01, K02 }, \
+    { K03, K04, K05 }, \
+    { K06, K07, K08 }, \
+    { K09, K0A, K0B }, \
+}
+
+#define KEYMAP LAYOUT
```

Comment:
We're actually trying to get rid of all instances of KEYMAP, there's no need to `#define` it here

```suggestion

```